bemani/backend/popnhell/popnhello.py

```diff
@@ -0,0 +1,137 @@
+# vim: set fileencoding=utf-8
+import binascii
+import copy
+import base64
+from collections import Iterable
+from typing import Any, Dict, List, Sequence, Union
+
+from bemani.backend.popnhell.base import PopnHelloBase
+from bemani.backend.ess import EventLogHandler
+from bemani.common import ValidatedDict, GameConstants, VersionConstants, Time
+from bemani.data import UserID
+from bemani.protocol import Node
+
+class HelloPopnMusic(
+    EventLogHandler,
+    PopnHelloBase,
+):
+    name = "Hello! Pop'n Music"
+    version = VersionConstants.HELLO_POPN_MUSIC
+
+    @classmethod
+    def handle_game_common_request(self, request: Node) -> Node:
+        #game_common
+        root = Node.void('game')
+
+        flag = Node.void('flag')
+        root.add_child(flag)
+
+        flag.set_attribute(id, '1')
+        flag.set_attribute(s1, '1')
+        flag.set_attribute(s2, '1')
+        flag.set_attribute(t, '1')
+
+        root.add_child(Node.u32(cnt_music, 36))
+        
+
+        return root
+    
+    def handle_game_shop_request(self, request: Node) -> Node:
+        #game_shop
+        root = Node.void('game')
+
+        return root
+
+    def handle_game_new_request(self, request: Node) -> Node:
+        #game_new
+        root = Node.void('game')
+        name = "0"
```

Comment:
Why are these here, and why not in the validated dict? Why do they even have to be in the default profile at all? get_int will return a "0" for anything not in the profile already, this is entirely useless. Also, I realize that that's the case where you did this for gfdm as well. Best to remove it.